src/rgw/rgw_common.h

```diff
@@ -383,16 +383,16 @@ class RGWEnv {
   void init(CephContext *cct);
   void init(CephContext *cct, char **envp);
   void set(const boost::string_ref& name, const boost::string_ref& val);
-  const char *get(const char *name, const char *def_val = NULL);
-  int get_int(const char *name, int def_val = 0);
+  const char *get(const char *name, const char *def_val = NULL) const;
```

Comment:
As we're already touching the line it might be worth to switch to `nullptr` as well.